packages/extension-ui/src/Popup/AuthManagement/WebsiteEntry.tsx

```diff
@@ -28,7 +28,7 @@ function WebsiteEntry ({ className = '', info, toggleAuth, url }: Props): React.
   return (
     <div className={`${className} ${info.isAllowed ? 'allowed' : 'denied'}`}>
       <div className='url'>
-        {url}
+        {info.origin} ({url})
```

Comment:
**Optional change:** that improves the website listing when using extensions (extension url is the extension id which means nothing to the user)